### DISCUSSION TITLE
CHange label font size

### DISCUSSION THREAD

**Discussion by ghost** [Mapsui/Mapsui#2543] [Q&A]
Hello...

I am new, very new, with Mapsui and net MAUI C# for mobile devices.

I am creating a mobile application for field work where, in addition to the graphic and alphanumeric information generated on office PCs, I can add whatever I now want in the field, basically photos, draw new plots, etc.
I'm pretty far along, but now I'm trying to add some functionality so the user has some leeway with what they see on the screen.

Among them is the ability to change the size of the labels you are viewing.

When creating the label layer I set a default size, but now I am not able to indicate a new size upon user request...

I attach a screenshot... the white labels with which I want to change size at the user's request

![image](https://github.com/Mapsui/Mapsui/assets/155519824/139349a9-828e-45cb-8b56-be2567c05d85)

Somebody can help me?

Thank you very much and forgive my ignorance, I am not a programmer, just self-taught.

**Comment by pauldendulk** [comment]
You initialize the layer with a LabelStyle. You should keep a reference to that LabelStyle in your code and assign it a new Font.

```csharp
void OnUserRequestedFontSizeChange(int fontSize)
{
    labelStyleForPlotsLayer.Font = new Font { FontFamily = "Verdana", Size = fontSize };
    map.Refresh();
}
```
